### ISSUE TITLE
Capitalization in dockerRepo map causes build failures

### ISSUE THREAD

**Issue by mfulgo** [nebula-plugins/nebula-docker-plugin#16]
If you specify a `dockerRepo` with capital letters in it, things break...
```gradle
nebulaDocker {
  // other settings here...
  dockerRepo = [
    devRepo: 'myrepository.example.com/myapp'
  ]
}
```
Will fail the build with
```
* What went wrong:
A problem was found with the configuration of task ':dockerTagImageDevrepo'.
> No value has been specified for property 'repository'.
```
However, the following works (note the all-lowercase):
```gradle
nebulaDocker {
  // other settings here...
  dockerRepo = [
    devrepo: 'myrepository.example.com/myapp'
  ]
}
```

**Comment by liviutudor** [comment]
Confirming this is a bug -- I can see the code is using lowercase all the time when accessing the `dockerRepo`. 
I can't remember any historical reason behind this (if there were any) so I'm going to make a change to support upper/lower case too.

**Comment by liviutudor** [comment]
Looking in the code I see why we are lowercasing it: we use the key of the map in creating the `push..` and `tag...` tasks: this key is lowercase'd and capitalized -- so in your example `devRepo` becomes `Devrepo` and then based on this we generate these tasks:

* `dockerTagImageDevrepo`
* `pushImageDevrepo`

I personally like the camelcasing of the tasks generated so I will allow mixed case in the map you signalled but also keep the capitalization in the tasks created.

**Comment by liviutudor** [comment]
fixed in 1.4.2
